### ISSUE TITLE
508 compatibility for DataTable sort

### ISSUE THREAD

**Issue by MSAJJAN-EEOC** [primefaces/primeng#4046]
**I'm submitting a ...**  
```
[ x] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```
**Current behavior**
  Sort is not accessible by keyboard and JAWS

**Expected behavior**
 Jaws should be able to read sort icon next to the column header and it should be accessible by keyboard to toggle the sorting order.

**What is the motivation / use case for changing the behavior?**
Blind people cannot understand when the icons are read by JAWS

* **Angular version:**  4.1.0

* **PrimeNG version:** 4.1.3

* **Browser:** 
 Chrome Latest | Firefox Latest | IE 11

**Comment by cagataycivici** [comment]
p-dataTable is deprecated and will be removed in favor of the new p-table (aka TurboTable) of 5.1.0 so closing the issue. Please try the new p-table once 5.1.0 is released.

**Comment by kkilton** [comment]
p-table is still not ADA compliant. Specifically, the sort icons and paginator arrows do not have labels and I cannot add aria-label's to them.

**Comment by MSAJJAN-EEOC** [comment]
I think it's fixed in 5.2.6.  Use the below code
`<p-sortIcon ariaLabelAsc=“text here” ariaLabelDesc=“text here”></p-sortIcon>`
